src/commands/run-command.yml

```diff
@@ -1,21 +1,22 @@
 description: >
-  Execute a MATLAB script, function, or statement. MATLAB exits with exit code 0 if the specified
-  script, function, or statement executes successfully without error. Otherwise, MATLAB terminates
-  with a nonzero exit code, which causes the build to fail. You can use the assert or error
-  functions in the command to ensure that builds fail when necessary. When you use this command,
-  all of the required files must be on the MATLAB search path.
+  Execute a MATLAB script, function, or statement. The command uses the topmost MATLAB version
+  on the system path.
```

Comment:
This sentence about the path exists only for this command, not the other two commands. Is this intentional? If the sentence should be added to the other sections too, then maybe a better idea is to add it to @orb.yml instead.